### DISCUSSION TITLE
Feedback on windows 10

### DISCUSSION THREAD

**Discussion by SilvioGrosso** [RawLabo/QuickRawPicker#11] [Ideas]
Hello qdwang,

First off, thanks a lot for releasing this software.
I have just tested it on Windows 10.
Tried on many images: NEF (nikon d700 - d850) and TIFFs
It works great: no crash at all so far :-)

Just a few proposals:
- add an option to rotate the image (even a simple shortcut to rotate 90° would suffice). I take my RAWs pictures on a tripood and a macro rail slider (for focus stacking stuff) and with QuickRawPicker they are all upside down now (180° off). Therefore, it would be great to being able to rotate them :-)
- add an option to zoom on the mouse cursor position. At present, the zoom always occurs on the center of the images. With other softwares (Darktable, ART, Gimp, Krita etc) the zoom occurs exactly at the mouse's cursor place on the image (e.g. on the left upper side).

Once again, thanks a lot. 
Aside from these minor remarks of mine your software looks extremely interesting and stable on Windows 10

**Comment by qdwang** [comment]
Thanks for you advices, I've created two enhancement issues in the issue list. 

https://github.com/qdwang/QuickRawPicker/issues/12
https://github.com/qdwang/QuickRawPicker/issues/13
